styles/Error.module.scss

```diff
@@ -0,0 +1,17 @@
+.canvas {
+    @apply flex flex-col size-full h-screen;
+}
+
+.page {
+    @apply flex flex-1 rounded-t-lg p-4 bg-[#222];
```

Comment:
Can you add #222 to the typescript config as a color (feel free to message on discord if you need more explanation)